src/operator/batch_norm.cc

```diff
@@ -77,21 +528,23 @@ Both ``gamma`` and ``beta`` are learnable parameters. But if ``fix_gamma`` is tr
 then set ``gamma`` to 1 and its gradient to 0.
 
 )code" ADD_FILELINE)
-.add_argument("data", "NDArray-or-Symbol", "Input data to batch normalization")
-.add_argument("gamma", "NDArray-or-Symbol", "gamma array")
-.add_argument("beta", "NDArray-or-Symbol", "beta array")
-.add_arguments(BatchNormParam::__FIELDS__());
+  .add_argument("data", "NDArray-or-Symbol", "Input data to batch normalization")
```

Comment:
Don't change this kind of formatting.
It was consistent with the format elsewhere. Your new format is not.
Also it introduces unnecessary merge conflicts for other people